src/styles/blocks/compare-bikes.scss

```diff
@@ -0,0 +1,49 @@
+.compare-bikes {
+  color: $color-white;
+
+  &__products {
+    @include page-grid;
+    row-gap: 40px;
+
+    @include on-tablet {
+      row-gap: 48px;
+    }
+  }
+
+  &__product {
+    grid-column: 1 / -1;
+
+    @include on-tablet {
+      grid-column: 2 /-2;
+    }
+
+    @include on-desktop {
+      grid-column: span 4;
+    }
+  }
+}
+
+.product {
+  &__title {
+    font-weight: 700;
+    font-size: 24px;
+    line-height: 33.6px;
+    margin: 1;
+    padding-bottom: 8px;
+  }
+
+  &__description {
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 22.4px;
+    margin: 1;
+    padding-bottom: 16px;
+  }
+
+  &__price {
+    font-weight: 700;
+    font-size: 24px;
+    line-height: 33.6px;
```

Comment:
just a tip for irl projects, for this one this is irrelevant: avoid using half pixels (they will generally be rounded, so round them)